.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,61 @@
+name: Feature Request ✨
+description: Suggest a feature
+title: "[Feature Request] "
+labels: enhancement
+body:
+  - type: checkboxes
+    id: existing-issue
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for this feature.
+      options:
```

Comment:
@Brijeshthummar02 , it's same here! Ask the link to the existing issue... if it's provided, so it exists, otherwise, the issuer can just let it blank...